plugins/flexibleengine/src/main/java/org/eclipse/xpanse/orchestrator/plugin/flexibleengine/FlexibleEngineTerraformResourceHandler.java

```diff
@@ -48,9 +48,8 @@ public void handler(DeployResult deployResult) {
         List<DeployResource> deployResourceList = new ArrayList<>();
         TfState tfState;
         try {
-            var stateFile = deployResult.getProperty().get("stateFile");
+            var stateFile = deployResult.getDeployedResultProperties().get("stateFile");
```

Comment:
I suggest that we move these codes to the terraform deployment, not in the plugins.